src/backend/marsha/core/tests/test_api_organization.py

```diff
@@ -0,0 +1,81 @@
+"""Tests for the Organization API of the Marsha project."""
```

Comment:
I would say, as long as you open an API endpoint, you need to test all verbs and types of users to ensure you are not leaking anything you don't intend to. On the model of what is done for existing endpoints?

Reply:
Ok then, I'll add more tests there